### PR TITLE
[sw] Replace hello_usbdev test with usbdev-only test

### PR DESCRIPTION
**Pull request by lenary**
This has been really flaky in CI, and in general we have been trying to
keep examples separate from tests, so it seems reasonable to disable.

The reason not to is it's the only end-to-end test we have covering
usbdev at the moment. I'm not sure about how covered this module is with
other kinds of tests (including DV).

Signed-off-by: Sam Elliott <selliott@lowrisc.org>

### CODE REVIEW

**Comment by lenary** [comment]
Ok! I'll leave this PR open while we wait for a response, but I have no intention of merging it as is.

**Comment by lenary** [comment]
I just pushed an update to not call `uart_init` yet again (it's done before `test_main` is called).

**Comment by lenary** [comment]
FWIW, I'm happy with this PR, but I cannot approve it - Pirmin also wrote part of the PR, so it may be we should prudently have a review from someone who's not us two.

**Comment by vogelpi** [comment]
> I just pushed an update to not call `uart_init` yet again (it's done before `test_main` is called).

Thanks, I wasn't aware of that.

File: hw/dv/dpi/usbdpi/usbdpi.c [hunk 1]
```diff
@@ -529,9 +529,9 @@ void pollRX(struct usbdpi_ctx *ctx, int sendHi, int nakData) {
       ctx->data[1] = 0x82;
       ctx->data[2] = 0 | CRC5(0x82, 11) << 3;
       ctx->data[3] = USB_PID_DATA0;
-      ctx->data[4] = 0x48;
-      ctx->data[5] = 0x69;
-      ctx->data[6] = 0x21;
+      ctx->data[4] = 0x48;  // "H"
```
Comment (mcy) [inline hw/dv/dpi/usbdpi/usbdpi.c:532]: Couldn't this just be `ctx->data[4] = 'H';`?

Comment (lenary) [reply hw/dv/dpi/usbdpi/usbdpi.c:532]: It felt easier to leave the packet assembly using hex constants as are used everywhere else in the file, but in this case provide some more documentation as to what the packet did.

**Pull request merged**